src/lib/qnio/nio_server.c

```diff
@@ -412,6 +430,17 @@ qns_server_start(char *node, char *port)
                         nioDbg("Accepted connection on descriptor %d "
                                   "(host=%s, port=%s)\n", infd, hbuf, sbuf);
                     }
+                    if(cmn_ctx->ssl_ctx)
+                    {
+                        ssl = SSL_new(cmn_ctx->ssl_ctx);
+                        SSL_set_fd(ssl, infd);
+                        ret = SSL_accept(ssl);
```

Comment:
Ok. I will take care of this in the vxhs server implementation. The server implementation in libqnio is a sample meant only to validate the client function independently. It need not have all the features that the vxhs IO head would have. VxHS does not use the server implementation from libqnio.